ReSharper.FSharp/src/FSharp.Psi.Features/src/Internal/FSharpInternalActionGroup.fs

```diff
@@ -42,12 +47,37 @@ type DumpCurrentFcsProjectAction() =
                     match fcsProjectProvider.GetFcsProject(psiModule) with
                     | Some fcsProject -> writer.WriteLine(fcsProject.TestDump(writer))
                     | _ -> ())
+            
+            
+[<Action("FSharp_Internal_DumpCurrentFile", "Dump current file")>]
+type DumpCurrentFileAction() =
+
+    [<DefaultValue>]
+    val mutable myTextControl: JetBrains.TextControl.ITextControl
+
+    interface IExecutableAction with
+        member this.Update(context, _, _) =
+            isNotNull (context.GetData(ProjectModelDataConstants.PROJECT))
```

Comment:
Why does the action become available when there's a project in the context, but checks a source file and an editor context during the actual execution? This way it's possible to call it without any file and to not see any results.

Reply:
Sorry, stupid newbie mistake. I hope the check for EDITOR_CONTEXT is the right one.